src/web/app/pages-session/session-submission-page/__snapshots__/session-submission-page.component.spec.ts.snap

```diff
@@ -985,7 +985,7 @@ exports[`SessionSubmissionPageComponent should snap with feedback session questi
                 </div>
               </div>
               <div
-                class="col"
+                class="px-3"
```

Comment:
These seems a little off. Seems like just moving the padding from the styles.scss here. Could you do a quick check to see if changing this to container works.

By right, not supposed to use grid system without enclosing container (or components that use container internally). See https://getbootstrap.com/docs/5.0/layout/containers/.
<img width="806" alt="Screenshot 2023-03-08 at 10 25 25 PM" src="https://user-images.githubusercontent.com/60355570/223738544-d756ec1c-d5f7-41f8-9f9e-16eb76397744.png">

Reply:
Good catch! Changing this to `container` indeed works.